website/docs/configuring_variants.md

```diff
@@ -29,13 +29,32 @@ Click on `create` and your webpush variant is ready!
 ## Android
 	AEROGEAR-10142
 ## iOS
-### APNs Certificate
+
  The following step-by-step guide gives you an introduction on how to use the AeroGear UnifiedPush Server for sending 
  Push Notifications to your own iOS Apps. The guide assumes you have an Apple developer account already setup, and uses tools available in macOS.
 
-#### Apple App ID and SSL Certificate for APNs
- Before you are able to use Push Notifications on your iOS Application, you must configure the Apple Push Notification service (APNs) with a certificate. 
+ To be able to send _push notifications_ to iOS, an _iOS variant_ must be created. That variant can be an _iOS (Certificate) Variant_ or an _iOS (APNS Token) Variant, 
+ depending on the way we are going to authenticate to the APNs: 
+
+ * with an APNs Certificate
+ * with an APNs Token
+
+ The first one is the legacy one: with the method you will require to have one certificate for each application and, furthermore,
```

Comment:
maybe change "with the method you will require to have " to  "with this method you will be required to have"

Reply:
👍 Done!